src/components/OpportunityCard/OpportunityCard.js

```diff
@@ -337,7 +395,7 @@ export default function OpportunityCard(props) {
                   icon={suitabilityScore > 75 ? <DoneIcon /> 
                     : suitabilityScore > 50 && suitabilityScore < 75 ? <HelpOutlineIcon />
                     : suitabilityScore < 50 ? <BlockIcon />
-                    : <BlockIcon />}
+                    : null}
```

Comment:
replace this with : <BlockIcon />}

the null causes the NaN on the card we saw before.